dist/leaflet.css

```diff
@@ -423,6 +423,9 @@ svg.leaflet-image-layer.leaflet-interactive path {
 .leaflet-control-attribution a:focus {
 	text-decoration: underline;
 	}
+.leaflet-control-attribution svg {

```

Comment:
We could give it a unique class, but how often do people use SVGs in their attribution, and change their `display`?

Reply:
(as this would override other SVGs in the attribution that have their `display` set to other than `inline`)